src/lighteval/commands/__init__.py

```diff
@@ -0,0 +1,21 @@
+# MIT License
```

Comment:
Is this file required?

Reply:
This file is not required, as of python 3.3+, it isn't necessary to have an __init__.py file for python to recognize the directory as a package. I just put it there to follow Pythonic packaging conventions, and to handle future commands. I don't know how complex the commands could get in the future, maybe there would be initialization logics that need to be defined for commands, I don't know. So I put it there just in case it could be useful in the future.